llvm/test/CodeGen/AMDGPU/lower-module-lds-precise-allocate-to-module-struct.ll

```diff
@@ -1,5 +1,5 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --function-signature --check-globals
-; RUN: opt -S -mtriple=amdgcn-- -passes=amdgpu-lower-module-lds --amdgpu-lower-module-lds-strategy=module < %s | FileCheck %s
+; RUN: opt -S -mtriple=amdgcn-- -passes=amdgpu-lower-module-lds < %s | FileCheck %s
```

Comment:
This is unfortunate - the precommit test has strategy module written at the top and shouldn't do, but it doesn't seem worth rerunning CI to fractionally simplify this diff